### ISSUE TITLE
Running a training job goes straight to failure

### ISSUE THREAD

**Issue by cosmicBboy** [gridai/gridai#127]
## 🐛 Bug

I have a training job that I've been using over the last few weeks that recently just started failing (I haven't changed any of the code on my end)

https://platform.grid.ai/#/runs?runName=8d817f7e-bf03-400d-a671-e5c0cdbf5603&nodetails=true

### Expected behavior

The job should at least be queued (?)

There are also no build/runtime logs to point me to the right direction.

### Current behavior

Goes straight to failure status

### To Reproduce

Command:
```
grid run \
--config /tmp/gridai-config-gpu.yml \
--ignore_warnings movenet/trainer.py \
--dataset /opt/datastore \
--n_epochs 5 \
--batch_size 2 \
--learning_rate 0.0003 \
--input_channels 64 \
--residual_channels 64 \
--layer_size 3 \
--stack_size 3 \
--checkpoint_every 1 \
--model_output_path models \
--pretrained_model_path /artifacts/gay-elion-8642-exp0/models/model.pth \
--pretrained_run_exp_name gay-elion-8642-exp0
```

config:

```
compute:

  provider:
    credentials: ${GRID_PROVIDER_CREDENTIALS}
    region: us-east-1
    vendor: aws

  train:
    disk_size: 300
    instance: p3.2xlarge
    cpus: 3
    gpus: 1
    memory: 60G
    framework: torch

    datastore_name: ${GRID_DATASTORE_NAME}
    datastore_version: ${GRID_DATASTORE_VERSION}
    datastore_mount_dir: ${GRID_DATASTORE_MOUNT_DIR}

    actions:
      on_image_build:
        - apt-get update -y
        - apt-get install wget libsndfile-dev -y
        - pip install lightning-grid

      on_before_training_start:
        - grid login --username ${GRID_USERNAME} --key ${GRID_API_KEY}
        - grid artifacts ${GRID_ARTIFACTS_RUNS_OR_EXPERIMENTS} --download_dir /artifacts || echo 'skip artifacts download'
        - echo "data store contents" && ls /opt/datastore/**/*
```

### Environment

 - OS (e.g., Windows): MacOS
 - Browser (e.g., Chrome, Safari): Chrome
 - CLI version: v0.3.92

**Comment by sunitaprakash** [comment]
Hello @cosmicBboy 

Thank you for reaching out to us. Our team will look into your ticket and get back to you. 

Regards,
Sunita Prakash

**Comment by sunitaprakash** [comment]
@cosmicBboy Just following up to see if you are still seeing this issue ?? Sometime ths might because of some network failure.

**Comment by cosmicBboy** [comment]
hi @sunitaprakash yes, I'm still seeing this issue

**Comment by cosmicBboy** [comment]
I got it to work now, looks like I needed to do the following things:
- make sure the branch I was using to launch the training run was pushed to remote git. It makes sense that this should be the case, though it would be nice to get a warning/error that a local git branch won't work
- since I use [yaml configs](https://docs.grid.ai/products/run-run-and-sweep-github-files/yaml-configs) I was missing the new (?) `dependency_file_info` config. It would be great it there was some validation of these config files to make sure required keys are enforced.
